aces/Loris
# [Core] Panel component updates

- [x] Add collapsed prop to allow users to set initial collapse status (open/closed)
- [x] Add panel header, only if the title is set, otherwise default to a simple panel
- [x] Add to `webpack`, because somehow it was missing

>**To test:**
>1. Add `collapsed={true|false}` prop to `Panel` component in `FilterForm`
>2. Open `dicom_archive` and make sure selection filter is in the desired state
>3. Enjoy! ☕️
